wgsl/index.bs

```diff
@@ -3243,11 +3249,11 @@ then:
 
 <div class='example wgsl function-scope' heading="[SHORTNAME] Valid loop if-break from a continuing clause">
   <xmp>
-    const a : i32 = 2;
+    const step : i32 = 1;
```

Comment:
This shouldn't need to be moved. The `step` should be visible inside the continue block.

Reply:
Another bug in tint then. Will fix.

Reply:
Done